chandra_aca/tests/test_all.py

```diff
@@ -94,6 +94,21 @@ def test_mag_for_p_acq():
     assert np.allclose(mag, 10.821, rtol=0, atol=0.01)
 
 
+def test_halfwidth_adjustment():
+    for mag, mults in ((6.0, [1.0, 1.0, 1.0, 1.0]),
+                       (8.5, [1.0, 1.0, 1.0, 1.0]),
+                       (9.25, [0.9, 0.8, 0.7, 0.505]),
+                       (10.0, [0.8, 0.6, 0.4, 0.01]),
+                       (11.0, [0.01, 0.01, 0.01, 0.01])):
+        pacq = acq_success_prob(mag=mag, halfwidth=120)
+        p160 = acq_success_prob(mag=mag, halfwidth=160)
+        p180 = acq_success_prob(mag=mag, halfwidth=180)
+        p240 = acq_success_prob(mag=mag, halfwidth=240)
+        p280 = acq_success_prob(mag=mag, halfwidth=280)
```

Comment:
Though technically, I suppose 280 would count as a value not called out explicitly in the logic.